.github/workflows/unit_tests.yml

```diff
@@ -11,7 +11,8 @@ jobs:
           - 3.0.4
           - 3.1.2
           - 3.2.0
-          - git+https://github.com/sphinx-doc/sphinx.git@3.2.x
+          # 3.2.x tip is currently broken (as of 11/05/2020)
+          #- git+https://github.com/sphinx-doc/sphinx.git@3.2.x
```

Comment:
Looks like 3.2.x was removed upstream and repaced with 3.3.x, perhaps we can try replacing it with 3.3.x? https://github.com/sphinx-doc/sphinx/branches

Reply:
Yeah, this was OK.